clustered/common/src/main/java/org/ehcache/clustered/common/ServerStoreCompatibility.java

```diff
@@ -42,27 +42,28 @@ public void verify(ServerStoreConfiguration serverConfiguration, ServerStoreConf
 
     PoolAllocation serverPoolAllocation = serverConfiguration.getPoolAllocation();
     PoolAllocation clientPoolAllocation = clientConfiguration.getPoolAllocation();
```

Comment:
It feels to me like the methods of ServerStoreCompatibility should really be instance methods of the various types it validates.  Does that make sense?